mappings/net/minecraft/server/world/TickingTracker.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/unmapped/C_dnmsfwrb net/minecraft/server/world/TickingTracker
```

Comment:
This needs some work. I don't like `TickingTracker`, but yarn's `SimulationDistanceLevelPropagator` is no better

Reply:
Understandable, something along the lines of SimulationDistanceTickTracker? or do we wish to move away form both namings?

Reply:
`ChunkTickTracker` maybe

Reply:
Yeah I guess, would then a package change be required?